terraform/services/api-waf-sync/main.tf

```diff
@@ -22,10 +22,10 @@ module "api_waf_sync_function" {
   schedule_expression = "cron(0/10 * * * ? *)"
 
   environment_variables = {
-    ENV             = var.env
-    APP_NAME        = "dpc-${var.env}-api-waf-sync"
-    WAF_IP_SET_NAME = "DPC_${upper(var.env)}_Implementer_IP_Set"
```

Comment:
Will WAF_IP_SET_NAME no longer be needed? If it is, a map of env to name in the locals could be a good solution.

Reply:
We don't use it any more

Reply:
Neither our lambda nor DPCs seemed to need it so we got rid of it.  Both our lambdas are using ~ `ipSetName := fmt.Sprintf("bcda-%s-api-customers", os.Getenv("ENV"))`.

Reply:
@gsf @gfreeman-navapbc  It also occurs to me that our DB_URL contains all the DB info, user, pw, host, instance, etc, is that problematic?

Reply:
Good point to raise. Terraform should mark the data as sensitive but to ensure the value is redacted in the logs it's better to accept the value as a variable and retrieve it in the workflow with aws-params-env-action. See an example of that at https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/0b23d0fa5e6978eae850ae88c1fe2f284e2a8f50/.github/workflows/api-rds-plan.yml#L40

Reply:
@gsf Ok, let me know if how I implemented this in the intended way.  Also one question this brings up is that (as far as I can tell) the terraform plan/apply happens only in dev env, which would grab the dev version of the parameter store, and pass that through to tf?  Which sounds problematic when it comes to other envs?